CMakeLists.txt

```diff
@@ -1241,6 +1241,7 @@ set(multilib_yaml_content "")
 # the real flash.
 add_library_variant(
     aarch64
+    SUFFIX exceptions_rtti
```

Comment:
If the order of variants with and without exceptions is important, it would be nice to have a comment on that.

Reply:
Thanks, I missed it.